Adding Deployment Quickstart (#9985)
PrefectHQ/prefect: docs/index.md

</figure>

#### New to Prefect?

> Head to our [First Steps](/getting-started/first_steps/) page to run your first basic flow and connect to Prefect's API.

Reviewer: ```suggestion
Head to the [Quick Start](/getting-started/first_steps/) to create, schedule, and run your flows.
```

Reviewer: Let's call the quick first thing we want people to do the Quickstart.

> If you're ready to dive in and learn Prefect, start with the [tutorial](/tutorial/) for an introduction to Prefect's core components and then checkout out the [concepts](/concepts/index/) for more in depth information.

Reviewer: If we want folks doing the the Quickstart (the old deployment quickstart) we may want to put it right on this first page. Then, after that we suggest tutorial, concepts, and guides. Or we keep prefect-quickstart on another page but we make that the emphasized place to go.

> Alternatively, to deploy a flow in as few steps as possible, check out our [deployment quickstart](/getting-started/deployment_quickstart/).

Reviewer: I thought we were folding this in with first-steps?

> For discussions of best practices with specific tools, explore our [guides](guides/index/) to see common use-cases. <div style="height: 10px"></div>

Reviewer: ```suggestion
For discussions of best practices with specific tools, explore our [guides](guides/index/) for common use-cases. <div style="height: 10px"></div>
```

[Concepts](/concepts){ .md-button .main-button--secondary .full } [Tutorial](/tutorial/){ .md-button .md-button--primary .main-button--primary .full }  [Guides](guides){ .md-button .main-button--secondary .full }

<div style="height: 10px"></div>
<p>Alternatively, read on for a quickstart of Prefect. </p>
---

## Prefect Quickstart

### Step 1: [Install Prefect](/getting-started/installation/)

<div class="terminal">
```bash
pip install -U prefect
```
</div>

> 

Reviewer: ```suggestion
See the [install guide](/getting-started/installation/) for more detailed installation instructions.
```

I would leave this link in case a user has any issues with install.

### Step 2: Connect to Prefect's API

> Sign up for a forever free [Prefect Cloud Account](/cloud/) or, alternatively, host your own [server](/host/) with a subset of Prefect Cloud's features.

Reviewer: ```suggestion
Sign up for a forever free [Prefect Cloud account](/cloud/) or, alternatively, host your own [server](/host/) with a subset of Prefect Cloud's features.
```

1. To sign in with an existing account or register an account, go to [https://app.prefect.cloud/](https://app.prefect.cloud/).

> 2. [Create a workspace](/cloud/workspaces/#create-a-workspace) for your account.

Reviewer: ```suggestion
2. If setting up a new account, [create a workspace](/cloud/workspaces/#create-a-workspace) for your account.
```

3. Use the `prefect cloud login` Prefect CLI command to [log into Prefect Cloud](/cloud/users/api-keys) from your environment.

    <div class="terminal">
    ```bash
    prefect cloud login
    ```
    </div>

### Step 3: Create a flow
**The fastest way to get started with Prefect is to add a `@flow` decorator to any python function**.

!!! tip "Rules of Thumb"
    - At a minimum, you need to define at least one flow function.
    - Your flows can be segmented by introducing task (`@task`) functions, which can be invoked from within these flows.
    - A task represents a discrete unit of Python code, whereas flows are more akin to parent functions accommodating a broad range of workflow logic.
    - Flows can be called inside of other flows (we call these subflows) but a task **cannot** be run inside of another task or from outside the context of a flow.

> Here is an example flow called `Get Repo Info` that contains 2 tasks:

Reviewer: ```suggestion
Here is an example flow named `Repo Info` that contains two tasks:
```
